scipy/sparse/tests/test_base.py

```diff
@@ -5549,31 +5554,33 @@ def check(cls, method_name):
         check(cls, method_name)
 
 
+@pytest.mark.thread_unsafe
```

Comment:
This is because these tests are very slow, not that they are all unsafe, right?

Reply:
There are some tests that employ warning suppression